lib/state/ui/reducer.ts

```diff
@@ -7,6 +7,22 @@ import * as T from '../../types';
 
 const emptyList: unknown[] = [];
 
+const dialogs: A.Reducer<T.DialogType[]> = (state = [], action) => {
+  switch (action.type) {
+    case 'CLOSE_DIALOG':
+      return state.slice(0, -1);
+
+    case 'SHOW_DIALOG':
+      return [...state, action.dialog];
+
+    case 'App.authChanged':
+      return [];
```

Comment:
this case looks weird. is this a result of signing out?
is there no sign-out action that's run?

Reply:
I actually don't know, I moved it over from here: https://github.com/Automattic/simplenote-electron/blob/develop/lib/flux/app-state.ts#L29

Reply:
yeah okay there it is - https://github.com/Automattic/simplenote-electron/commit/0c033413fec2dc050c68eec8ac849a764978068f

so I guess that what happens is you sign out, and if we don't clear it, then the app re-opens with the dialogs still open. I'm ear-marking that for a better action-type like `SIGN_OUT`, which I presume we'll get when we refactor `App.authChanged`